pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -144,21 +145,43 @@ public PhiManifest generateDataForReport(@NonNull ElasticSearchParticipantDto pa
         phiManifest.setOrderStatus(mercuryOrderDto.getOrderStatus());
         phiManifest.setOrderStatusDate(DateTimeUtil.getDateFromEpoch(mercuryOrderDto.getStatusDate()));
 
-        String pediatricResponse = convertActivityAnswerValue((Boolean) participant.getParticipantAnswerInSurvey(
-                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_STABLE_ID, SOMATIC_CONSENT_TUMOR_PEDIATRIC_QUESTION).orElse(false));
-        phiManifest.setSomaticConsentTumorPediatricResponse(pediatricResponse);
-        String assentAddendumResponse = convertActivityAnswerValue((Boolean) participant.getParticipantAnswerInSurvey(
-                CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_STABLE_ID, SOMATIC_ASSENT_ADDENDUM_QUESTION).orElse(false));
+        String pediatricResponse = convertBooleanActivityAnswerToReportValue(participant.getParticipantAnswerInSurvey(
```

Comment:
Instead of converting to a boolean in this method, we pass the full `Optional` into a new method that knows how to deal with blank values and return a String that can be rendered in the manifest.